src/direct.js

```diff
@@ -1,23 +1,30 @@
 const net = require('net');
 
 /**
- * @param {http.ClientRequest} request
- * @param {net.Socket} source
- * @param {buffer.Buffer} head
- * @param {*} handlerOpts
- * @param {*} server
+ * @typedef Options
```

Comment:
Currently this is duplicated but will be a single interface in the TypeScript PR.